course_content/matplotlib_intro.ipynb

```diff
@@ -113,7 +113,7 @@
      "source": [
       "## Working with the axes\n",
       "\n",
-      "As has already been mentioned, most of your time building a visualisation in matplotlib will be spent on the Axes artist. Whilst the documentation for the Axes artist is very detailed, it is also rather difficult to navigate (this is an ongoing area of improvement).\n",
+      "As has already been mentioned, most of your time building a graphic in matplotlib will be spent on the Axes artist. Whilst the matplotlib documentation for the Axes artist is very detailed, it is also rather difficult to navigate (though this is an area of ongoing improvement).\n",
```

Comment:
Curious that graphic is better than visualisation. I'm not too fussed either way though.